docs/reference/setup/install/targz-start.asciidoc

```diff
@@ -1,5 +1,17 @@
 ==== Running Elasticsearch from the command line
 
+[IMPORTANT]
+.Unidentified developer warnings
+====
+If using macOS Catalina or later macOS versions, you may receive an unidentified
+developer warning when you first run {es}. Unless turned off, this warning
+prevents you from running {es}.
+
```

Comment:
I think we should provide a little more context to indicate that Elastic software has been submitted for notarization and we're working to resolve any issues related to the shifting requirements Just recommending that users ignore "unauthorized developer" warnings seems a bit sketchy. Maybe something along the lines of:

"Apple's roll-out of stricter notarization requirements affected the notarization of the {version} {es} artifacts.  If macOS Catalina displays an unidentified developer warning when you first run {es}, you will need to add a security override to run this version. This will be resolved in a subsequent release."

Reply:
Thanks @debadair. I addressed this in aa418a4a1e9.